float32[] pose
---
float32[] focus_pnt
